cli/cmd/start/start.go

```diff
@@ -50,28 +51,35 @@ func StartCmd(cfg *config.Config) *cobra.Command {
 					return fmt.Errorf("required arg <path> missing")
 				}
 
-				fmt.Println("No existing project found. Enter name to initialize a new Rill project.")
-				questions := []*survey.Question{
-					{
-						Name: "name",
-						Prompt: &survey.Input{
-							Message: "Enter project name",
-							Default: cmdutil.DefaultProjectName(),
-						},
-						Validate: func(any interface{}) error {
-							name := any.(string)
-							if name == "" {
-								return fmt.Errorf("empty name")
-							}
-							return nil
-						},
-					},
-				}
-
 				if !cmd.Flags().Changed("example") {
-					if err := survey.Ask(questions, &projectPath); err != nil {
+					currentDir, err := filepath.Abs("")
+					if err != nil {
+						return err
+					}
+
+					displayPath := currentDir
+					defval := true
+					projectPath = currentDir
```

Comment:
nit: move `displayPath` and `defval` to above the `strings.Contains` check that first uses them